.github/workflows/ci.yml

```diff
@@ -91,14 +133,86 @@ jobs:
+      - name: Install oneAPI Compiler (classic)
+        if: matrix.cxx_compiler == 'icpc'
+        run: sudo apt install -y intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic
       - name: Run testing
         shell: bash
         run: |
           set -x
           sudo rm -rf /opt/intel/oneapi/dpl || true
           source /opt/intel/oneapi/setvars.sh intel64 || true
+          if [[ "${{ matrix.cxx_compiler }}" == "dpcpp" ]]; then
+            echo "::warning::$(dpcpp --version)"
+          fi
+          if [[ "${{ matrix.backend }}" == "dpcpp" ]]; then
+            make_targets="sycl_iterator.pass"
+            ctest_flags="-R sycl_iterator.pass"
+            echo "::warning::dpcpp backend is set. Compile and run only sycl_iterator.pass"
+          fi
           mkdir build && cd build
           cmake -DCMAKE_CXX_STANDARD=${{ matrix.std }} -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
             -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }} -DONEDPL_BACKEND=${{ matrix.backend }} -DONEDPL_DEVICE_TYPE=${{ matrix.device_type }} ..
-          make VERBOSE=1 -j${BUILD_CONCURRENCY}
-          ctest --timeout ${TEST_TIMEOUT} --output-on-failure
+          make VERBOSE=1 -j${BUILD_CONCURRENCY} ${make_targets}
+          ctest --timeout ${TEST_TIMEOUT} --output-on-failure ${ctest_flags}
+
+  windows-testing:
+    name: ${{ matrix.os }}_${{ matrix.cxx_compiler }}_backend=${{ matrix.backend }}_device=${{ matrix.device_type }}_cxx${{ matrix.std }}_${{ matrix.build_type }}
+    runs-on: ['${{ matrix.os }}']
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: windows-2019
+            cxx_compiler: dpcpp
+            std: 17
+            build_type: release
+            backend: tbb
+            device_type: HOST
+          - os: windows-2019
+            cxx_compiler: cl
+            std: 17
+            build_type: release
+            backend: tbb
+            device_type: HOST
+          - os: windows-2019
+            cxx_compiler: dpcpp
+            std: 17
+            build_type: release
+            backend: dpcpp
+            device_type: CPU
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install oneTBB
+        shell: cmd
+        run: |
+          call %CONDA%/condabin/conda.bat activate base
+          conda install -c intel tbb-devel
+      - name: Install oneAPI Compiler
+        if: matrix.cxx_compiler == 'dpcpp' || matrix.cxx_compiler == 'icx'
+        shell: cmd
+        run: |
+          call %CONDA%/condabin/conda.bat activate base
+          conda install -c intel dpcpp_win-64
+      - name: Run testing
+        shell: cmd
+        run: |
+          IF "${{ matrix.backend }}" == "dpcpp" (
+            set ninja_targets=sycl_iterator.pass
+            set ctest_flags=-R sycl_iterator.pass
+            echo ::warning::dpcpp backend is set. Compile and run only sycl_iterator.pass
+          )
+          call %CONDA%/condabin/conda.bat activate base
```

Comment:
Is it intentionally between two `if`'s? The `if`s are about the same area (dpcpp), consider writing them near each other to increase readability

Reply:
Fixed